src/util/kafka/utility.js

```diff
@@ -0,0 +1,300 @@
+/*****
```

Comment:
Can we not call this `src/util/kafka/config.js`?

Reply:
It isnt just config, its produces and commits messages as well as the breadcrumb path

Reply:
What has the breadcrump path have to do with Kafka?